services/web-app/components/resource-card/resource-card.js

```diff
@@ -41,8 +46,13 @@ const ResourceCard = (props) => {
     }
   )
 
+  // if aspectRatio is not specified and it's a card with title displaying at Lg breakpoint,
+  // default aspectRatio to 16:9, all other cases default to 2:1
```

Comment:
Let's double check on this, I think we might want to default to 16:9 across the board at large, 2:1 at xlg.